recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - python  # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
   host:
-    - scip
+    - scip =8.1.0
```

Comment:
Is current `pyscipopt` broken with `8.1.0`  (broken environments) ? 
In this case, we'll want to patch to add an upper bound to previous release.

Or it is pinned to a given version (the previous) and you want to rebuild with 8.1.0 ?
In this case, we'll may want to add a build matrix/variant to have pyscipopt available with multiple versions.